test/extended/openstack/local-bd-etcd.go

```diff
@@ -0,0 +1,162 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/flavors"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/stretchr/objx"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+)
+
+const minEtcdDiskSizeGiB = 7
+const etcdBlockDeviceName = "etcd"
+
+var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack platform", func() {
+	defer g.GinkgoRecover()
+
+	var computeClient *gophercloud.ServiceClient
+	var ctx context.Context
+	var dc dynamic.Interface
+	var controlPlaneFlavor string
+	var clusterCPMS objx.Map
+	var clientSet *kubernetes.Clientset
+
+	g.BeforeEach(func() {
+		ctx = context.Background()
+
+		g.By("preparing a dynamic client")
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dc, err = dynamic.NewForConfig(cfg)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		clientSet, err = e2e.LoadClientset()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		computeClient, err = client(serviceCompute)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
+		skipUnlessEtcdAdditionalBlockDevice(ctx, dc)
+
+	})
+
+	g.It("checks resources when etcd is on an ephemeral block device", func() {
```

Comment:
This will render as:
> "[sig-installer][Suite:openshift/openstack] The OpenStack platform checks resources when etcd is on an ephemeral block device

which makes little sense. Can you please rephrase in terms of what you expect to be in the cluster?